[BUG] [beta] Using PDF Editor with "Create new documents" and "Delete original" gives error

### Description

Using PDF Editor with "Create new documents" and "Delete original" gives the following error when clicking on "Proceed" in the dialog:

```json
{"headers":{"headers":{},"normalizedNames":{},"lazyUpdate":null},"status":400,"statusText":"Unknown Error","url":"https://paperless-beta.host.example/api/documents/edit_pdf/","ok":false,"redirected":false,"responseType":"basic","name":"HttpErrorResponse","message":"Http failure response for https://paperless-beta.host.example/api/documents/edit_pdf/: 400 ","error":"Error performing PDF edit, check logs for more detail."}
```

I have validated that on 2.20.15 this works (same files, same db, same host system, nearly identical docker-compose).

### Steps to reproduce

1. Choose a document
2. Use Actions >> PDF-Editor 
3. Manipulate at least on PDF page
4. Use the options "Create new document(s)" and check "Delete original"
5. Click "Proceed"

**Observed Behavior**
An error toast is shown. The error is above in the description.

### Webserver logs

```bash
[2026-05-07 14:21:00,756] [INFO] [paperless.bulk_edit] Editing PDF of document 2427 with 1 operations

[2026-05-07 14:21:00,794] [INFO] [paperless.bulk_edit] Released archive serial numbers for documents []

[2026-05-07 14:21:00,821] [INFO] [paperless.bulk_edit] Restored archive serial numbers for documents []

[2026-05-07 14:21:00,822] [ERROR] [paperless.bulk_edit] Error editing document 2427: Starting chords requires a result backend to be configured.

Note that a group chained with a task is also upgraded to be a chord,

as this pattern requires synchronization.

Result backends that supports chords: Redis, Database, Memcached, and more.

Traceback (most recent call last):

  File "/usr/src/paperless/src/documents/bulk_edit.py", line 857, in edit_pdf

    ).apply_async(

      ^^^^^^^^^^^^

  File "/usr/local/lib/python3.12/site-packages/celery/canvas.py", line 2156, in apply_async

    return self.run(tasks, body, args, task_id=task_id, kwargs=kwargs, **merged_options)

           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "/usr/local/lib/python3.12/site-packages/celery/canvas.py", line 2249, in run

    app.backend.apply_chord(

  File "/usr/local/lib/python3.12/site-packages/celery/backends/base.py", line 811, in apply_chord

    self.ensure_chords_allowed()

  File "/usr/local/lib/python3.12/site-packages/celery/backends/base.py", line 1202, in ensure_chords_allowed

    raise NotImplementedError(E_CHORD_NO_BACKEND.strip())

NotImplementedError: Starting chords requires a result backend to be configured.

Note that a group chained with a task is also upgraded to be a chord,

as this pattern requires synchronization.

Result backends that supports chords: Redis, Database, Memcached, and more.

[2026-05-07 14:21:00,823] [WARNING] [paperless.api] An error occurred performing PDF edit: An error occurred while editing the document: Starting chords requires a result backend to be configured.

Note that a group chained with a task is also upgraded to be a chord,

as this pattern requires synchronization.

Result backends that supports chords: Redis, Database, Memcached, and more.
```

### Browser logs

```bash

```

### Services logs

```bash

```

### Paperless-ngx version

3.0.0-beta.rc1

### Host OS

DSM 7.3

### Installation method

Docker - official image

### System status

```json

```

### Browser

Firefox / Zen Browser

### Configuration changes

_No response_

### Please confirm the following

- [x] I believe this issue is a bug that affects all users of Paperless-ngx, not something specific to my installation.
- [x] This issue is not about the OCR or archive creation of a specific file(s). Otherwise, please see above regarding OCR tools.
- [x] I have already searched for relevant existing issues and discussions before opening this report.
- [x] I have updated the title field above with a concise description.

--- a/src/paperless/settings/__init__.py
+++ b/src/paperless/settings/__init__.py
@@ -650,6 +650,11 @@ logging.config.dictConfig(LOGGING)
 # https://docs.celeryq.dev/en/stable/userguide/configuration.html
 
 CELERY_BROKER_URL = _CELERY_REDIS_URL
+CELERY_RESULT_BACKEND = _CELERY_REDIS_URL
+CELERY_RESULT_SERIALIZER = "signed-pickle"
+# Results are only needed for chord synchronization
+# a short TTL avoids Redis memory accumulation.
+CELERY_RESULT_EXPIRES = 3600
 CELERY_TIMEZONE = TIME_ZONE
 
 CELERY_WORKER_HIJACK_ROOT_LOGGER = False

